tests/App/DataFixtures/BookFixtures.php

```diff
@@ -28,17 +28,17 @@ final class BookFixtures extends Fixture implements DependentFixtureInterface
 
     public function load(ObjectManager $manager): void
     {
-        $author = $this->getReference(AuthorFixtures::AUTHOR);
+        $author = $this->getReference(AuthorFixtures::AUTHOR, Author::class);
         \assert($author instanceof Author);
```

Comment:
All the assert are useless now
https://phpstan.org/r/4e6bc4f6-2837-4f78-875d-a1b59205597c

Reply:
I can try it again, I tried before and couldnt remove them.

Reply:
```
  34     Parameter #3 $author of class Sonata\DoctrineORMAdminBundle\Tests\App\Entity\Book constructor expects
         Sonata\DoctrineORMAdminBundle\Tests\App\Entity\Author|null, object given.
  35     Parameter #1 $category of method Sonata\DoctrineORMAdminBundle\Tests\App\Entity\Book::addCategory() expects
         Sonata\DoctrineORMAdminBundle\Tests\App\Entity\Category, object given.
  43     Parameter #1 $category of method Sonata\DoctrineORMAdminBundle\Tests\App\Entity\Book::addCategory() expects
         Sonata\DoctrineORMAdminBundle\Tests\App\Entity\Category, object given.
  48     Parameter #1 $category of method Sonata\DoctrineORMAdminBundle\Tests\App\Entity\Book::addCategory() expects
         Sonata\DoctrineORMAdminBundle\Tests\App\Entity\Category, object given.
```